config/local.yaml

```diff
@@ -1,5 +1,9 @@
 watcher:
   router-refresh-min-interval-s: 10
-  kubernetes-service-domain-sufix: mystack.com
-  mystack-controller-domain: controller.mystack.com
-  mystack-logger-domain: logger.mystack.com
+postgres:
+  host: postgres
+  user: mystack_controller
```

Comment:
we should not have the database as dependency here, we can just grab de domains from annotations in the app services